main/src/main/java/org/mobilitydata/gtfsvalidator/util/HtmlOutputUtil.java

```diff
@@ -0,0 +1,199 @@
+      notices.append(
+          noticeFrame(noticeCode, noticeSeverity, totalNotices, sampleColumns, sampleRows));
+    }
+    String outputHeader =
+        outputHeader(
+            (noticesQty.get(ERROR) + noticesQty.get(WARNING) + noticesQty.get(INFO)),
+            noticesQty.get(ERROR),
+            noticesQty.get(WARNING),
+            noticesQty.get(INFO));
+    return outputFrame(
+        jqueryDependency, styleCss, scriptJs, outputHeader, notices.toString(), outputFooter());
+  }
+
+  public static String resourceLoader(String resourcePath) {
+    String content = "";
+    try (InputStream stream =
+        HtmlOutputUtil.class.getClassLoader().getResourceAsStream(resourcePath)) {
+      Scanner scanner = new Scanner(stream).useDelimiter("\\A");
+      content = scanner.hasNext() ? scanner.next() : "";
+    } catch (IOException e) {
+      System.out.println("The resource was not found at \"" + resourcePath + "\".");
+    }
+    return content;
+  }
+
+  protected static String noticeColumnsBuilder(ArrayList<String> noticeFields) {
+    StringBuilder columns = new StringBuilder("                ");
+    for (String field : noticeFields) {
+      columns.append("<th>").append(field).append("</th>");
+    }
+    columns.append("\n");
+    return columns.toString();
+  }
+
+  protected static String noticeRowsBuilder(
+      ArrayList<String> noticeFields, JsonArray sampleNotices) {
+    StringBuilder rows = new StringBuilder();
+    for (JsonElement notice : sampleNotices) {
+      JsonObject noticeJson = notice.getAsJsonObject();
+      rows.append("              <tr>\n");
+      for (String field : noticeFields) {
+        rows.append("                <td>")
+            .append(noticeJson.get(field).toString())
+            .append("</td>\n");
+      }
+      rows.append("              </tr>\n");
+    }
+    return rows.toString();
+  }
+
+  protected static String outputHeader(int total, int errors, int warnings, int infos) {
+    return "<h1>VALIDATION REPORT</h1>\n"
+        + "<h2>"
+        + total
+        + " notices reported ("
+        + errors
+        + " errors, "
+        + warnings
+        + " warnings, "
+        + infos
+        + " infos)"
+        + "</h2>\n"
+        + "<p>Please visit <a href=\"https://github.com/MobilityData/gtfs-validator/blob/master/docs/NOTICES.md\">NOTICES.md</a> and <a href=\"https://github.com/MobilityData/gtfs-validator/blob/master/RULES.md\">RULES.md</a>.</p>\n"
+        + "<table class=\"accordion\">\n"
+        + "  <thead>\n"
+        + "    <tr>\n"
+        + "      <th>Code</th><th>Severity</th><th>Total</th>\n"
+        + "    </tr>\n"
+        + "  </thead>\n"
+        + "  <tbody>\n";
+  }
+
+  protected static String outputFooter() {
+    return "  </tbody>\n"
+        + "</table>\n"
+        + "<p>This validation report was generated using the <a href=\"https://github.com/MobilityData/gtfs-validator\">GTFS Schedule validator</a>.</p>\n";
```

Comment:
I added
- the arguments passed to the validator, which includes the dataset input name
- the date and time
- the validator version

I did not add the GTFS dataset version for now since `feed_info.txt` is optional. @isabelle-dr, thoughts?